tree/10_ncf_internals/modules/templates/jinja2-templating.py

```diff
@@ -66,6 +68,24 @@ def render(opts, args):
     if opts.strict:
         env.undefined = StrictUndefined
 
+    # Register customs
+    sys.path.append('/var/rudder/ncf/local/modules/templates/')
```

Comment:
the full path for the local template should be  /var/rudder/ncf/common/10_ncf_internals/modules/templates

Reply:
You are right but the path provided at line 72 is the path of the custom filter module (aka `jinja2_custom`) and it must be outside of any overwritten mechanism.

Reply:
To be copied to /var/rudder/ncf/local/modules/templates/ you want to say ?

Reply:
yes, i made a typo, sorry